packages/monaco/src/browser/monaco-editor-service.ts

```diff
@@ -42,6 +42,9 @@ export class MonacoEditorService extends monaco.services.CodeEditorServiceImpl {
     @inject(EditorManager)
     protected readonly editors: EditorManager;
 
+    @inject(PreferenceServiceImpl)
```

Comment:
please program against an interface, not an implementation, i.e. `PreferenceService`